astropy/coordinates/tests/test_sky_coord.py

```diff
@@ -601,6 +601,16 @@ def test_position_angle():
     assert cicrs.position_angle(cfk5) > 90.0 * u.deg
     assert cicrs.position_angle(cfk5) < 91.0 * u.deg
 
+    # regression check for bug in #5702
+    cfk5B1950 = SkyCoord(1*u.deg, 0*u.deg, frame='fk5', equinox='B1950')
```

Comment:
Still think that each of the functions that was fixed needs a test (e.g. using the backwards/forwards equivalence method using for position angle).